AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -728,7 +723,7 @@ public boolean onQueryTextChange(String newText) {
     }
 
 
-    private void displaySyncBadge(Menu menu) {
+    protected void displaySyncBadge(Menu menu) {
```

Comment:
I think if you are just opening access for testing you could flag it as such (and the default is private? there are examples of usage scattered all throughout the code)

```suggestion
    @VisibleForTesting
    protected void displaySyncBadge(Menu menu) {
```

Reply:
Yes, okay! Totally forgot to add the annotation